python/pacemaker/resource.py

```diff
@@ -0,0 +1,22 @@
+"""A module for managing cluster resources."""
+
+__all__ = ["list_standards"]
+__copyright__ = "Copyright 2025 the Pacemaker project contributors"
+__license__ = "GNU Lesser General Public License version 2.1 or later (LGPLv2.1+)"
+
+import libxml2
+
+import _pcmksupport
+from pacemaker.exceptions import PacemakerError
+
+
+def list_standards():
+    """Return a list of supported resource standards."""
+    try:
+        xml = _pcmksupport.list_standards()
+    except _pcmksupport.PacemakerError as e:
+        raise PacemakerError(*e.args) from None
+
+    doc = libxml2.xmlDoc(xml)
+
+    return [item.getContent() for item in doc.xpathEval("/pacemaker-result/standards/item")]
```

Comment:
The intention here (and in any further functions) would be to convert the returned XML into a python native type, like a list of strings here.  That's what I'm going through so much trouble everywhere to deal with the XML types.